app/src/main/res/values/strings.xml

```diff
@@ -564,6 +570,8 @@ New dictionary:
     <!-- User dictionary settings. Hint for the text field to type the optional shortcut to add to the user dictionary. [CHAR LIMIT=35] -->
     <!-- This resource is corresponding to msgid="7333763456561873445" -->
     <string name="user_dict_settings_add_shortcut_hint">Optional shortcut</string>
+    <!-- User dictionary settings. Hint for the text field to type the frequency to add to the user dictionary. [CHAR LIMIT=35] -->
+    <string name="user_dict_settings_add_frequency_hint">Type a frequency between 0–255</string>
```

Comment:
If internals like frequency are exposed to the user, some sort of description should be provided, i.e. what is the effect of the chosen value, and maybe some explanation of special values (I think 0 and 14 are treated a little different, don't remember details though)

Reply:
OK, I'll add a text 👍

FYI : I haven't noticed any influences in relation to the special values.
For example, with a frequency of 0, the word is still proposed, regardless of whether the "Block offensive word" option is activated or not.

What I'm sure of, having tested with the words (which don't exist at all in French) "bkgzui" (frequency 100) and "bkgzuitha" (frequency 250), is that the word "bkgzuitha" is proposed before the other.

Knowing all this, do I still need to mention special values in the explanations?

Reply:
> For example, with a frequency of 0, the word is still proposed, regardless of whether the "Block offensive word" option is activated or not.

Thanks for testing. Possibly the user dictionary behaves different here than the main dictionary.

I think some explanation would be good, possibly also a different name. _Frequency_ is reasonable for a dictionary where the _frequency_ value is derived from actual word count, but for the user dictionary this does not apply.

Reply:
"Suggestion value" is a more appropriate term in this case.

Reply:
What do you think of the term "Weight"?

As an explanation, we can say:

"_The weight determines whether a word should be more or less suggested; this means that the higher its value, the more the word will be suggested and corrected.
This value must be between 0 and 255._"

Reply:
Weight sounds fine.
When I suggested an explanation I didn't think about where to actually put it, do you have any idea?
If not maybe it could be omitted... not sure.

Reply:
I'm not sure what to say... one more translation is better than several open issues on this subject.

It's up to you whether I take it off or not.

Reply:
Hmmm, it's a rather minor thing that I'd rather have in some sort of _info_ / FAQ file that should be done anyway before the app is properly released.

Reply:
Great idea too!
I won't add any more explanations.